src/Adapter/MailTemplate/MailTemplateTwigRenderer.php

```diff
@@ -67,12 +71,14 @@ class MailTemplateTwigRenderer implements MailTemplateRendererInterface
     public function __construct(
         Environment $twig,
         LayoutVariablesBuilderInterface $variablesBuilder,
-        HookDispatcherInterface $hookDispatcher
+        HookDispatcherInterface $hookDispatcher,
+        ConfigurationInterface $configuration
```

Comment:
This is a backward compatibility break because if someone is currently using this constructor, after your PR is merged it will now crash (fatal error) because the new constructor argument will be missing in his code.

But good news: next version is a major version! BC breaks are allowed 😄  . However we need to list them in the PR description.

Reply:
By the way PHPStan (see below) reported issues _exactly_ because of this problem: in some unit tests, MailTemplateTwigRenderer is being constructed. You did not update the code of these tests so these tests now crash.

Reply:
My opinion is that there is not a lot of value to be tested here. You give `$configuration` to the class, then it uses `get()` to retrieve `giftWrapping` and then we expose it as a smarty variable. We do not transform the data, we just transfer it.

Reply:
Yes. Existing unit tests need to be updated so that they pass